connaisseur/tests/integration/integration-test.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+set -euo pipefail
+
+envsubst <connaisseur/tests/integration/values.yaml >helm/values.yaml
+
+make install || { echo "Failed to install Connaisseur"; exit 1; }
+echo "Successfully installed Connaisseur"
+
+kubectl create -f - << EOF >output.log 2>&1 || true
```

Comment:
maybe use `kubectl run pod --image=connytest/testimage:unsigned`. it's a lot smaller and should suffice for the test. same for the signed one. note that the output messages will change.